packages/shared/styles/components/molecules/SfSelect.scss

```diff
@@ -45,7 +45,8 @@
   &__option {           
     @include background(
       --select-option-background,
-      var(--c-white)
+      var(--_select-option-background-image),
+      var(--_select-option-background-color, var(--c-white))
```

Comment:
I think color should go first :slightly_smiling_face: what do you think?

Reply:
I think it works, as I found example with image first on mdn, but I should follow our convention which is color first, so you are right about this :)